xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1044 @@
+      a directory <emphasis>.obs</emphasis> which contains a file called
+      <emphasis>workflows.yml</emphasis>.</para>
+
+      <para>The content of <emphasis>.obs/workflows.yml</emphasis> could look
+      like this:<screen>main_workflow:
+  steps:
+    - branch_package
+        source_project: OBS:Server:Unstable
+        source_package: obs-server
+        target_project: OBS:Server:Unstable:CI
+  filters:
+    event: pull_request
+rebuild_master:
+  steps:
+    - rebuild_package
+        project: home:Admin
+        package: ctris
+  filters:
+    event: push
+    branches:
+      only:
+        - master</screen></para>
+
+      <para>The file can contain multiple workflows, but each one needs an
+      <emphasis>unique</emphasis> name. The previous example contains two
+      workflows: <emphasis>main_workflow</emphasis> and
+      <emphasis>rebuild_master</emphasis>.</para>
+
+      <para>OBS assumes the configuration file is placed in your target branch
+      and not in other branches or forks.</para>
+
+      <note>
+        <title>Workflows Cleanup</title>
+
+        <para>Workflows are cleaned up after themselves when pull requests are
+        closed or merged. Projects in which you branched or linked packages
+        will be deleted, the same thing for the repositories you configured
+        for those projects.</para>
+
+        <para>If you decide to reopen the pull request, this will also be
+        handled by workflows. Your projects, their branched/linked packages
+        and their configured repositories will be restored.</para>
+      </note>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.steps">
+        <title>Steps</title>
+
+        <para>Currently, the supported steps are:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Branch a Package (branch_package).</para>
+          </listitem>
+
+          <listitem>
+            <para>Link a Package to a Project (link_package).</para>
+          </listitem>
+
+          <listitem>
+            <para>Configure Repositories/Architectures for a Project
+            (configure_repositories)</para>
+          </listitem>
+
+          <listitem>
+            <para>Rebuild a Package (rebuild_package)</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>Some of these steps need to get the sources from the SCM pull
+        request. Read <link
+        linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.steps.service">OBS
+        SCM Service</link> for more details.</para>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.branch_a_package">
+          <title>Branch a Package</title>
+
+          <para>Given we have a source package called<emphasis>
+          obs-server</emphasis> coming from a source project called
+          <emphasis>OSB:Server:Unstable</emphasis>, and a target project
```

Comment:
I think you should use easier project/package names. Using OBS packages to explain an OBS feature is way too much inception...

Reply:
![inception](https://media.giphy.com/media/7pHTiZYbAoq40/giphy.gif)